src/main/java/Project_16x16/App.java

```diff
@@ -197,13 +182,8 @@ public void setup() {
 	 */
 	private void load() {
 		Tileset.load(this);
-		if (new File("Assets/Font/font-pixel-48.vlw").exists()) {
```

Comment:
Why was font file check removed?

Reply:
resources in a JAR can't be accessed with ```File``` ...at least without specifying the full path to the jar, and loadFont already throws/dies with a descriptive error message if it can't find the font.